docs/release_notes.rst

```diff
@@ -1,6 +1,12 @@
 Release Notes
 =============
 
+v2.1.4
```

Comment:
API break so needs to be v3.0.0

(also add a commit that says:

```
Sem-Ver: api-break
```

at the end to force this version change)